### PR TITLE
Updated Discordapp.

### PR DESCRIPTION
**Pull request by byparallax**
* Edited name
* Added notes
* Updated domains

### CODE REVIEW

File: sites.json [hunk 1]
```diff
@@ -1666,11 +1666,14 @@
     },
 
     {
-        "name": "Discord",
+        "name": "Discord(app)",
```
Comment (Nightfirecat) [inline sites.json]: Why `Discord(app)` instead of `Discord`? They don't call it `Discordapp`, usually.

File: sites.json [hunk 2]
```diff
@@ -1666,11 +1666,14 @@
...
         "url": "https://support.discordapp.com/hc/en-us/articles/212500837-How-do-I-permanently-delete-my-account-",
         "difficulty": "easy",
+        "notes":"You will need to transfer the ownership of your servers.",
```
Comment (Nightfirecat) [inline sites.json]: How about `If you're a server owner, you'll need to either <a href="https://support.discordapp.com/hc/en-us/articles/213595197-How-do-I-delete-a-server-">delete the server</a>, or <a href="https://support.discordapp.com/hc/en-us/articles/216273938-How-do-I-transfer-server-ownership-">transfer ownership</a> for account deletion to succeed.` instead? (adapted from the URL)

File: sites.json [hunk 3]
```diff
@@ -1666,11 +1666,14 @@
...
         "domains": [
-            "discordapp.com"
+            "discordapp.com",
+            "ptb.discordapp.com",
```
Comment (Nightfirecat) [inline sites.json]: No need to add subdomains.

File: sites.json (hunk 2 above)
Comment (Nightfirecat) [reply sites.json]: Also, can a space be added after the colon, before the quote for the second string? eg. `"notes": " ... ",`

File: sites.json (hunk 1 above)
Comment (byparallax) [reply sites.json]: Both names are pretty commonly used

File: sites.json (hunk 2 above)
Comment (byparallax) [reply sites.json]: That sounds good to me; I updated the notes.

File: sites.json (hunk 3 above)
Comment (byparallax) [reply sites.json]: I removed the subdomains, then.

File: sites.json (hunk 1 above)
Comment (Nightfirecat) [reply sites.json]: The website exclusively refers to it as "Discord", as do its various legal pages.

**Comment by Nightfirecat** [comment]
I've chosen to go ahead with the `notes` addition, but not the `name` change. Thank you for your contribution!

**Pull request merged**